docs/metasploit-framework.wiki/Metasploit-Guide-MSSQL.md

```diff
@@ -26,6 +26,175 @@ use auxiliary/admin/mssql/mssql_sql
 run rhost=192.168.123.13 username=administrator password=p4$$w0rd sql='select auth_scheme from sys.dm_exec_connections where session_id=@@spid'
 ```
 
+### Logging in and obtaining a session
+To log in or obtain an interactive session on an MSSQL instance running on the target, use mssql_login
+
+```msf
+use auxiliary/scanner/mssql_login
+run CreateSession=true RPORT=1433 RHOSTS=192.168.2.242 USERNAME=user PASSWORD=password
+```
+
+The CreateSession option, when set to true, will result in returning an interactive MSSQL session with the target machine
+on a successful login:
+
+```msf
+[*] 192.168.2.242:1433    - 192.168.2.242:1433 - MSSQL - Starting authentication scanner.
+[!] 192.168.2.242:1433    - No active DB -- Credential data will not be saved!
+[+] 192.168.2.242:1433    - 192.168.2.242:1433 - Login Successful: WORKSTATION\user:password
+[*] MSSQL session 1 opened (192.168.2.1:60963 -> 192.168.2.242:1433) at 2024-03-15 13:41:31 -0500
+[*] 192.168.2.242:1433    - Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
+```
+
+Which you can interact with using `sessions -i <session id>` or `sessions -1` to interact with the most recently opened session.
+
+```msf
+msf6 auxiliary(scanner/mssql/mssql_login) > sessions
+
+Active sessions
+===============
+
+  Id  Name  Type   Information                     Connection
+  --  ----  ----   -----------                     ----------
+  1         mssql  MSSQL test @ 192.168.2.242:143  192.168.2.1:60963 -> 192.168.2
+                   3                               .242:1433 (192.168.2.242)
```

Comment:
For the docs, we try not to provide snippets that have been wraapped onto new lines - as when it's rendered on the website it won't look visually correct on larger screens 👀